packages/contracts-bedrock/contracts/dispute/FaultDisputeGame.sol

```diff
@@ -27,12 +27,12 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone {
     /**
      * @notice The current Semver of the FaultDisputeGame implementation.
      */
-    string internal constant VERSION = "0.0.1";
+    string internal constant VERSION = "0.0.2";
```

Comment:
nice

Reply:
We currently have a diff between these contracts and the other contracts where they are `Semver` and these are not. I'm not opposed to moving away from the immutable approach of `Semver` and just using `ISemver` but we need to standardize. This contract should be `ISemver`

Reply:
@tynes Can do in a separate PR 👍

Reply:
Can we put up a linear ticket for this? I think whenever we split work out into another pr, we should get in the habit of writing a linear ticket so we don't forget/others can pick up the work

Reply:
Created one [here](https://linear.app/optimism/issue/CLI-4145/featctb-inherited-semver-contract) :)